prj.conf

```diff
@@ -131,6 +131,10 @@ CONFIG_XEN_SHELL=y
 CONFIG_XRUN=y
 CONFIG_XRUN_SHELL_CMDS=y
 
+# Enable cpu power management
+
+CONFIG_PM_CPU_OPS=y
```

Comment:
Move to system components section. It is not related to xen.

Reply:
Done